opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java

```diff
@@ -128,9 +128,14 @@ private static void processRadioButtonsSpecialWidget(JSONObject widget, List<Str
         for (int i = 0; i < jsonArray.length(); i++) {
 
             JSONObject jsonObject = jsonArray.getJSONObject(i);
-
-            if (widget.has(JsonFormConstants.VALUE) && !TextUtils.isEmpty(widget.getString(JsonFormConstants.VALUE)) &&
-                    jsonObject.getString(JsonFormConstants.KEY).equals(widget.getString(JsonFormConstants.VALUE))) {
+            String widgetValue = null;
+            if(widget.has(JsonFormConstants.VALUE))
+             widgetValue = widget.getString(JsonFormConstants.VALUE);
+            if(widgetValue != null && widgetValue.startsWith("{"))
```

Comment:
Use the `!StringUtils.isBlank()` instead

Reply:
this file has been updated